scripts/pin_the_slow_way.py

```diff
@@ -31,19 +31,25 @@
 pinned = {
           'boost': 'boost 1.61.*',
           'bzip2': 'bzip2 1.0.*',
+          'ffmpeg': 'ffmpeg 2.8.*',
           'fontconfig': 'fontconfig 2.11.*',
           'freetype': 'freetype 2.6.*',
           'hdf5': 'hdf5 1.8.17|1.8.17.*',
           'icu': 'icu 56.*',
           'jpeg': 'jpeg 9*',
+          'libblitz': 'libblitz 0.10|0.10.*',
+          'libmatio': 'libmatio 1.5.*',
```

Comment:
Looks like `libmatio` might need to be pinned tighter than 1.5.*.

ref: https://abi-laboratory.pro/tracker/timeline/matio/